### DISCUSSION TITLE
Get validation loss during training

### DISCUSSION THREAD

**Discussion by EmanuelOverflow** [open-mmlab/mmengine#1483] [Q&A]
Hi everyone. I'm new to openmmlab. 

I seaeched around, but with no concrete results. I'm using openmmlab with config files. I'm unable to get the validation loss for each epoch. It is a value, that as you know, tells you more than the final accuracy. Is there any method to get this metric? It could be very useful to have plots overlapped, but it could be in a second time manually. I was trying to use a custom config for logprocessor, but in the ValLoop class it seems that loss is never computed...maybe it is necessary to write a LossMetric to pass to evaluator? It is very strange that this settings is not available by default.

**Comment by zhouzaida** [comment]
Hi, please see #1486
